libs/pika/async_mpi/include/pika/async_mpi/mpi_polling.hpp

```diff
@@ -26,17 +26,29 @@
 #include <vector>
 
 namespace pika::mpi::experimental {
+
+    enum stream_index : std::uint32_t
+    {
+        default_stream = 0,
+        send_stream,
+        receive_stream,
+        collective_stream,
+        user_stream,
+    };
+
     namespace detail {
         // -----------------------------------------------------------------
         // by convention the title is 7 chars (for alignment)
         using print_on = pika::debug::detail::enable_print<false>;
         static constexpr print_on mpi_debug("MPI_FUT");
+        struct mpi_queue;
```

Comment:
Is it a stream or a queue? I don't mind which one, but it should be used consistently.

Reply:
it's a queue of stuff that we call a stream! I'll rename the vars appropriately.